src/geometric_transformations.rs

```diff
@@ -317,6 +317,146 @@ where
     warp(image, &projection, interpolation, default)
 }
 
+/// Rotates an image 90 degrees clockwise.
+///
+/// # Examples
+/// ```
+/// use imageproc::geometric_transformations::rotate90;
+/// use imageproc::gray_image;
+///
+/// let image = gray_image!(
+///     1, 2, 0, 0;
+///     3, 4, 0, 0;
+///     0, 0, 0, 0;
+///     0, 0, 0, 0);
+///
+/// let rotated = rotate90(&image);
+///
+/// assert_eq!(rotated, gray_image!(
+///     0, 0, 3, 1;
+///     0, 0, 4, 2;
+///     0, 0, 0, 0;
+///     0, 0, 0, 0));
+/// ```
+pub fn rotate90<P>(image: &Image<P>) -> Image<P>
```

Comment:
I don’t feel strongly either way but it’s a bit unfortunate to use angles in degrees in the names of these functions when angles everywhere else in this file (/crate) are in radians. Can’t think of better names off the top of my head though.

Reply:
As some prior art, opencv just has a single `rotate()` function which takes an enum that looks like `ROTATE_90_CLOCKWISE`

Reply:
Maybe we should do the same and use a single function with an enum, but I don't feel particularly strongly either way.

Reply:
I wondered the same thing. I’ll just merge as is for now. Thanks.